presentation/components/src/test/common/ContentBuilder.test.ts

```diff
@@ -536,14 +642,44 @@ describe("ContentBuilder", () => {
           }],
         };
         const displayValues = {
-          [field.name]: undefined,
+          [rootField.name]: undefined,
         };
         const item = createTestContentItem({ values, displayValues });
-        const record = ContentBuilder.createPropertyRecord({ field, childFields: [{ field: nestedField }] }, item);
+        const record = ContentBuilder.createPropertyRecord({ field: rootField, childFields: [{ field: nestedField }] }, item);
+        expect({ field: record.field.name, record: record.record }).to.matchSnapshot();
+      });
+
+      it("creates record for empty nested content value", () => {
+        const nestedField = createTestNestedContentField({ name: "nested-field", nestedFields: [] });
+        const rootField = createTestNestedContentField({ name: "root-field", nestedFields: [nestedField] });
+        const values = {
+          [rootField.name]: [{
+            primaryKeys: [createTestECInstanceKey({ id: "0x1" })],
+            values: {
+              [nestedField.name]: [{
+                primaryKeys: [createTestECInstanceKey({ id: "0x2" })],
+                values: {
+                },
+                displayValues: {
+                },
+                mergedFieldNames: [],
+              }],

```

Comment:
Empty objects could be contracted

Reply:
why?

Reply:
Yes, the compiler doesn't care. It just looks like there was an intention for these objects to contain something.